syne_tune/optimizer/schedulers/searchers/dyhpo/hyperband_dyhpo.py

```diff
@@ -237,3 +237,9 @@ def summary_schedule_keys() -> List[str]:
     def summary_schedule_records(self) -> Dict[str, Any]:
         histogram = Counter([x[1] for x in self._schedule_records])
         return {name: histogram[value] for name, value in _SUMMARY_SCHEDULE_RECORDS}
+
+    def support_early_checkpoint_removal(self) -> bool:
```

Comment:
More difficult to support this, because trials can be paused anywhere. This is for future work.